.github/workflows/main.yml

```diff
@@ -0,0 +1,68 @@
+name: main
+
+on:
+  push:
+    branches: [master, gh-actions-with-upstream-branch]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python_version: [3.6, 3.7]
```

Comment:
I can't use much newer versions, because the tensorflow version we install doesn't have wheels for Py3.9 for instance. This can be updated in a separate PR.